scripts/pre_push_hook.py

```diff
@@ -295,6 +295,16 @@ def _install_hook():
         print 'Copied file to .git/hooks directory'
 
 
+def does_diff_include_js_files(files_to_lint):
+    """Returns true if diff includes JS files."""
```

Comment:
Missing Args, Returns docstrings.

/cc @oppia/dev-workflow-team we should check for these?

Reply:
Done.

Reply:
> Missing Args, Returns docstrings.
> 
> /cc @oppia/dev-workflow-team we should check for these?
Yeah, I also think this check should be added, that every function should have Args and Returns docstrings. What do you suggest @apb7?

Reply:
What about those functions which does not return anything nor has any args? One example of this is here -> https://github.com/oppia/oppia/blob/develop/core/controllers/base.py#L520

Reply:
Hm, the lint check should be able to distinguish those by looking at the function signature? Not sure if you can detect returns though. Maybe see if the body contains any return statements. 

(And yep, if there are no Args, then no "Args:" section is needed in the docstring. Ditto if there's nothing returned.)

Reply:
@seanlip We have issue #6081 open for this. Thanks